apps/myaccount/src/layouts/dashboard.tsx

```diff
@@ -292,6 +298,7 @@ export const DashboardLayout: FunctionComponent<PropsWithChildren<DashboardLayou
                             />
                         ) }
                     >
+                        <SystemNotificationAlert />
```

Comment:
Move this inside the  suspense component

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/8071/commits/f87fea4adf0003abe10f632137d68dafa2c1cc79